crates/foundry/evm/core/src/backend/mod.rs

```diff
@@ -479,7 +483,7 @@ impl Backend {
         };
 
         if let Some(fork) = fork {
-            let (fork_id, fork, _) = backend
+            let (fork_id, fork, _, fork_block_number) = backend
```

Comment:
It seems like `fork_block_number` can be extracted from `CreateFork`

Reply:
Fixed in https://github.com/NomicFoundation/edr/commit/74a47d17aa564b3169657b43397285f2ad62c31e